src/data/nav-items.yaml

```diff
@@ -41,12 +41,128 @@
 
 - title: Using ACD
   pages:
+    - title: "Overview"
+      path: /clouddocs/overview/
     - title: "Getting started"
       path: /usage/getting-started/
     - title: "Software Development Kits"
       path: /usage/sdks/
     - title: "Custom Resource APIs"
       path: /usage/custom-resource-apis/
+    - title: "FAQs"
+      path: /clouddocs/faqs/
+    - title: "Known Limitations"
+      path: /clouddocs/known_limitations/
+
+
+# - title: ACD Service Documentation
+#   pages:
+#     - title: "Getting started tutorial"
+#       path: /clouddocs/getting-started/
+#     - title: "Dependencies on other IBM Cloud services"
+#       path: /clouddocs/dependencies/
+- title: How to
+  pages:
+    - title: "Analyzing Text"
+      path: /clouddocs/analyze_text/
+    - title: "Customizing"
+      path: /clouddocs/customizing/
+    # - title: "Managing Access Roles"
+    #   path: /clouddocs/access_management/
+    # - title: "Securing your data in Annotator for Clinical Data"
+    #   path: /clouddocs/data-security/
+    # - title: "Activity Tracker events"
+    #   path: /clouddocs/at_events/
+- title: Annotators
+  pages:
+    - title: "Allergies (Depracated)"
```

Comment:
"Depracated" is misspelled here and a number of other places. Probably do a global search for that and correct it.

Reply:
resovled